### ISSUE TITLE
[Feature request] Change base image to kasmvnc

### ISSUE THREAD

**Issue by ilkersigirci** [jlesage/docker-czkawka#23]
### Idea

Recently, linuxserver webtop move to [kasm base image](https://github.com/linuxserver/docker-baseimage-kasmvnc) for better performance. Related blog post is [here](https://www.linuxserver.io/blog/webtop-2-0-the-year-of-the-linux-desktop). 

I think we can also profit from performance improvements if the base image will be changed.

**Comment by jlesage** [comment]
Yes I did see this.  Applications like Czkawka would probably not have any performance benefit, but other apps could.
I will investigate to see how heavy it would be to support kasm.  One problem I see is the lost of support for "standard" VNC clients.
